src/firefly/js/metaConvert/vo/DataLinkStandAloneConverter.js

```diff
@@ -0,0 +1,23 @@
+import {getDataLinkData} from '../../voAnalyzer/VoDataLinkServDef.js';
+import {createDataLinkSingleRowItem} from './DataLinkProcessor.js';
+
+/**
+ *
+ * @param {TableModel} table
+ * @param {DataProductsConvertType} converterTemplate
+ * @param {DataProductsFactoryOptions} options
+ * @return {DataProductsConvertType}
+ */
+export function makeDatalinkStaneAloneConverter(table,converterTemplate,options={}) {
+    if (!table) return converterTemplate;
+    return {...converterTemplate,
+        converterId: `DatalinkStandalone-${table.tbl_id}`};
+}
+
+export async function getDatalinkStandAlineDataProduct(table, row, activateParams, options) {
+    const dataLinkData= getDataLinkData(table);
+    const dlData= dataLinkData?.[row];
+
+    const item= createDataLinkSingleRowItem({dlData, activateParams, baseTitle:'datalink!!! data', options});
+    return item;
```

Comment:
should the `baseTitle` be something else? I'm guessing this was left over from debugging.

Reply:
yes, I will fix it.